docs/ref/migration-operations.txt

```diff
@@ -269,7 +269,7 @@ be removed (elided) when :ref:`squashing migrations <migration-squashing>`.
 RunPython
 ---------
 
-.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None, elidable=False)
+.. class:: RunPython(code, reverse_code=None, atomic=None, hints=None, elidable=False)
```

Comment:
I've added short paragraph with a link to non-atomic migrations. Originally, I had a decision matrix in there as well but I thought it would be more confusing than helpful to most people and I finally removed it again:

```
================  ================  ================  ===========
Can rollback DDL  Atomic migration  RunPython atomic  Transaction
================  ================  ================  ===========
Yes               *True*            *None*            Migration
Yes               *True*            True              Migration
Yes               *True*            False             Migration
Yes               False             *None*            No
Yes               False             True              Operation
Yes               False             False             No
No                *True*            *None*            Operation
No                *True*            True              Operation
No                *True*            False             No
No                False             *None*            No
No                False             True              Operation
No                False             False             No
================  ================  ================  ===========
```